Templates/BlobTrigger-Batch/run.bat

```diff
@@ -0,0 +1,4 @@
+echo OFF
+SET /p input=<%input%
+echo Windows Batch script processed blob '%input%'
+echo %input% > %output%
```

Comment:
I think this will only work for text files, and probably corrupt binaries. Why not simply `copy %input% %output%`?